test/README.md

```diff
@@ -0,0 +1,69 @@
+# test
+
+## This directory is intended to hold the orchestration, framework and utilities needed to test Tanzu Community Edition
+
+### contents
+
+1. Deployment Automation:
+
+    - [aws](https://github.com/vmware-tanzu/community-edition/tree/main/test/aws), [azure](https://github.com/vmware-tanzu/community-edition/tree/main/test/azure), [docker](https://github.com/vmware-tanzu/community-edition/tree/main/test/docker) and [vsphere](https://github.com/vmware-tanzu/community-edition/tree/main/test/vsphere) directories are home to deployment automation of Tanzu Community Edition on the respective providers.
+
+    - Usage:
+        - Environment variables:
+            - Make sure that the environment variables needed for different providers are exported.
+            - Here are the environment variables needed for [aws](https://github.com/vmware-tanzu/community-edition/blob/f2e47713ec0da0e9f68649f6e8f678c77bd26c24/test/aws/deploy-tce-managed.sh#L11), [azure](https://github.com/vmware-tanzu/community-edition/blob/f2e47713ec0da0e9f68649f6e8f678c77bd26c24/test/azure/deploy-management-and-workload-cluster.sh#L11) and [vsphere](https://github.com/vmware-tanzu/community-edition/blob/f2e47713ec0da0e9f68649f6e8f678c77bd26c24/test/vsphere/run-tce-vsphere-standalone-cluster.sh#L10).
+
+        - Run the following commands from the root of the repository to trigger deployment automation:
+            - Management Cluster deployment automation on:
+                - aws:
+
+                    ```shell
+                    make aws-management-cluster-e2e-test
+                    ```
+
+                - azure:
+
+                    ```shell
+                    make azure-management-and-workload-cluster-e2e-test
+                    ```
+
+                - docker:
+
+                    ```shell
+                    make tce-docker-managed-cluster-e2e-test:
+                    ```
```

Comment:
We could add vSphere management cluster make target too. The only tricky thing is, it will work only with appropriate configuration and / environment. We have to start documenting more around that I think

Reply:
#1554 adds the make target, maybe we can document the additional info in the README in vSphere dir and then point in the parent README.md?

Reply:
sure! 👍 Once this gets merged, we can make the change to this README.md in #1554 with link to the vSphere test README 👍